storage/backend/boltoption_solaris.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build solaris
+
+package backend
+
+import (
+	"syscall"
+
+	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/boltdb/bolt"
+)
+
+// syscall.MAP_POPULATE on Linux 2.6.23+ does sequential read-ahead
+// which can speed up entire-database read with boltdb. We want to
+// enable MAP_POPULATE for faster storage key-value store restore.
+var boltOptions = &bolt.Options{MmapFlags: syscall.MAP_POPULATE}
```

Comment:
That's how boltdb does. And since I specify the OS by file name suffix, we need to list both unix and solaris, I think. Do you have a better way? Actually, the file name suffix overrides build tags.